### PR TITLE
include specific helpers instead of database::#{type}

### PR DESCRIPTION
**Pull request by bkw**
This is because community cookbook database removed
database::mysql in v4.0.2, and database::postgresql
is a mere include of postgresql::ruby anyways.

In case of mysql, we now include mysql2_chef_gem
as directed by the database cookbook.
For postsgresql we just include postgresql::ruby
ourselves, since that may be removed next…

### CODE REVIEW

**Comment by zuazo** [comment]
Hello @bkw, thanks for your PR! :smile: 

So, if I understand you correctly, the `database` cookbook version in the metadata should be changed to be `~> 4.0`, right? If that's the case, can you change it?

**Comment by bkw** [comment]
Actually, it's more like the other way around. Without this patch, we'd have to make sure we only use database < 4.0.2. With this patch, it doesn't matter, since we moved the functionality behind database::$type into our own code base.

TLDR: no dependency change necessary ;-)

**Comment by zuazo** [comment]
Great! Thanks for the explanation.

**Comment by zuazo** [comment]
Released in [`1.4.0`](https://github.com/onddo/postfixadmin-cookbook/releases/tag/1.4.0).
